### ISSUE TITLE
Do you give a dictionary keyed by argument names or layers names to set_lr_mult ?

### ISSUE THREAD

**Issue by erogol** [dmlc/mxnet#2112]
I like to set learning rate of all feature layers to 0 and keep only the last later active for learning. For this purpose I ve done this. Is this the right way? I asked since I could not see my net improving at training time and I suspect the way of doing this.

  ```
 # set optimizer
    sgd = mx.optimizer.SGD(
        learning_rate      = args.lr,
        momentum           = 0.9,
        wd                 = 0.00001,
        **optimizer_args)

    # get all parameter list
    internals = network.get_internals()
    arg_names = internals.list_arguments()

    # set lr for all feature layers to 0
    lr_dict = dict()
    for arg_name in arg_names:
        lr_dict[arg_name] = 0

    # set only the last layer active learner
    lr_dict['fc1_new_weight'] = 1
    lr_dict['fc1_new_bias'] = 1
```

**Comment by piiswrong** [comment]
argument names

**Comment by erogol** [comment]
Thanks for the answer in a flash ... It seems correct then right ?

**Comment by chenjx1005** [comment]
@erogol  Did you find the reason that your net doesn't improve?
